.github/workflows/autofix.ci.yml

```
@@ -0,0 +1,54 @@
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install Frontend Dependencies
+        run: |
+          cd cur8t-web
+          pnpm install
+
+      # Setup Python for Python formatting
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)